src_OLD/ShortStackLib/Cmdlets/NewPullRequest.cs

```diff
@@ -0,0 +1,36 @@
+using System.Management.Automation;
+using System.Threading;
+
+namespace Microsoft.Tools.Productivity.ShortStack
+{
+    //---------------------------------------------------------------------------------
+    /// <summary>
+    /// Creates a pull request for the current stack.
+    /// </summary>
+    //---------------------------------------------------------------------------------
+    [Cmdlet(VerbsCommon.New, "PullRequest")] 
+    [OutputType(typeof(StackInfo))]
+    public class NewPullRequest : ShortStackCmdletBase
+    {
+        /// <summary>
+        /// If this is specified, any dangling work will be committed
+        /// with this description
+        /// </summary>
+        [Parameter(Position = 1)]
+        public string CommitDescription { get; set; }
+
+        //---------------------------------------------------------------------------------
```

Comment:
nit: are these comment lines necessary? :)  i find doc comments are significant visual breaks already

Reply:
Definitely not necessary.  I find that they relieve my stress levels as I'm scrolling around n the code.